lib/tasks/comms.rake

```diff
@@ -3,67 +3,66 @@
 require "csv"
 
 namespace :comms do
-  desc "Send the comms to the schools in the pilot"
-  task :send_pilot, [:path_to_csv] => :environment do |_task, args|
+  desc "Chase pilot schools"
```

Comment:
Maybe make this more descriptive? "Chase 2023 pilot schools that haven't set up their cohort" or something like that

Reply:
The description needed to be a bit more generic because we chase two types of pilot schools:
1. Schools with no 2023 training programmes
2. Schools with no SITs